### PR TITLE
PSD 220-221-222 (#156)

### PR DESCRIPTION
**Pull request by DimarrWS**
* PSD-220,221,222

* PSD-220,221,222

* Update ci.yml

* Update ci.yml

* PSD-220,221,222

* Update ci.yml

* PSD-220,221,222. Docker (no user root)

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -23,8 +23,9 @@ jobs:
           echo "TOOL_NAME=$TOOL_NAME" >> $GITHUB_ENV
           echo "RELEASE=false" >> $GITHUB_ENV
           echo "VERSION=0.0.0.ci0" >> $GITHUB_ENV
+          echo $GITHUB_REF
           echo "TOOL_DIR=$TOOL_NAME" >> $GITHUB_ENV
-          if [[ $GITHUB_REF == refs/tags/v* ]]; then
+          if [[ $GITHUB_REF == refs/tags/v* || "$GITHUB_REF" == *"refs/tags/test-v"* ]]; then
```
Comment (tidharm) [inline .github/workflows/ci.yml:28]: Syntax should be unified. Either `$GITHUB_REF == refs/tags/v* || "$GITHUB_REF" == refs/tags/test-v*` or `"$GITHUB_REF" == *"refs/tags/v"* || "$GITHUB_REF" == *"refs/tags/test-v"*`.
(I think it should be the former)

Comment (rammatzkvosky) [reply .github/workflows/ci.yml:28]: This one works:
`if [[ "$GITHUB_REF" == *"refs/tags/v"* || "$GITHUB_REF" == *"refs/tags/test-v"* ]]`

**Review by tidharm — CHANGES_REQUESTED** [review]
@DimarrWS please review inline comments.
Also, is this the version I installed and tested (22.12.4.1a4)? Or is this another increment?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (DimarrWS) [reply .github/workflows/ci.yml:28]: I'm using if [[ "$GITHUB_REF" == *"refs/tags/v"* || "$GITHUB_REF" == *"refs/tags/test-v"* ]] as @RAm provided me. But anyway, this is not the principal issue.

**Comment by DimarrWS** [comment]
> @DimarrWS please review inline comments. Also, is this the version I installed and tested (22.12.4.1a4)? Or is this another increment?

This version has another increment because I tried to solve the vulnerability issue (user root or not defined user). As I wrote yesterday - I will explain it on Monday. We need to understand how we want (if we really to invest to here) to solve it.

**Review by tidharm — APPROVED** [review]

**Review by DimarrWS — COMMENTED** [review]
Agree

**Comment by DimarrWS** [comment]
This violation "**Ensure that a user for the container has been created**" will be solved in refactoring branch

**Pull request merged**